src/ofxGLWarper.h

```diff
@@ -16,8 +16,8 @@
 class ofxGLWarper{
 	
 public:
-    enum CornerLocation{
-        TOP_LEFT = 0,
+    enum CornerID{
```

Comment:
Ok... Just BEFORE changing, what would you think of adding : 
```
// in ofxGLWarper.h
#define CornerLocation CornerID
```
I tested and it works when you have an app like:
`warper.setCorner(ofxGLWarper::CornerLocation::TOP_RIGHT,800, 200);`
And to be honest, if you just call like this, it works the same:
`warper.setCorner(ofxGLWarper::TOP_RIGHT,800, 200);`

Just a last try because I prefer this name, but I wouldn't fight ;)

Reply:
Right it is just an issue about taste, but as said, there's no need to change it and it will introduce possible backwards compatibility. I wouldn't add that define either.